patches/minecraft/net/minecraft/entity/Entity.java.patch

```diff
@@ -211,6 +214,19 @@
 +        return ItemStack.field_190927_a;
 +    }
 +
++    /**
++     * Called when a user uses the creative pick block button on this entity.
++     *
++     * @param target The full target the player is looking at
++     * @param player The player that picked this entity
++     * @return A ItemStack to add to the player's inventory, empty ItemStack if nothing should be added.
++     */
++    @javax.annotation.Nonnull
```

Comment:
This annotation can be removed, everything is Nonnull return value by default in vanilla minecraft code.